Extensions/TopDownMovementBehavior/topdownmovementruntimebehavior.ts

```diff
@@ -273,7 +286,7 @@ namespace gdjs {
 
       //Update the speed of the object
       if (direction != -1) {
-        directionInRad = (direction * Math.PI) / 4.0;
+        directionInRad = ((direction + this._movementAngleOffset / 45) * Math.PI) / 4.0;
         directionInDeg = direction * 45;
```

Comment:
Do you think this should be
```suggestion
        directionInDeg = direction * 45 + this._movementAngleOffset;
```

?

Reply:
directionInDeg is used to rotate the object and remember the angle of the latest movement. 
I think in both case they should be the real angle that was moved, like directionInRad. So we should add the movementAngleOffset.

(`this._angleOffset` is still useful to add an offset to the object (useful if your object is not orientated properly), but this is only for rendering, we should have both)